internal/db/rdb/mysql/mysql_test.go

```diff
@@ -2846,84 +2828,237 @@ func Test_mySQLClient_deleteMetaWithTx(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           ctx: nil,
-		           tx: nil,
-		           uuid: "",
-		       },
-		       fields: fields {
-		           db: "",
-		           host: "",
-		           port: 0,
-		           user: "",
-		           pass: "",
-		           name: "",
-		           charset: "",
-		           timezone: "",
-		           initialPingTimeLimit: nil,
-		           initialPingDuration: nil,
-		           connMaxLifeTime: nil,
-		           dialer: nil,
-		           dialerFunc: nil,
-		           tlsConfig: nil,
-		           maxOpenConns: 0,
-		           maxIdleConns: 0,
-		           session: nil,
-		           connected: nil,
-		           eventReceiver: nil,
-		           dbr: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           ctx: nil,
-		           tx: nil,
-		           uuid: "",
-		           },
-		           fields: fields {
-		           db: "",
-		           host: "",
-		           port: 0,
-		           user: "",
-		           pass: "",
-		           name: "",
-		           charset: "",
-		           timezone: "",
-		           initialPingTimeLimit: nil,
-		           initialPingDuration: nil,
-		           connMaxLifeTime: nil,
-		           dialer: nil,
-		           dialerFunc: nil,
-		           tlsConfig: nil,
-		           maxOpenConns: 0,
-		           maxIdleConns: 0,
-		           session: nil,
-		           connected: nil,
-		           eventReceiver: nil,
-		           dbr: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		func() test {
+			ctx, cancel := context.WithCancel(context.Background())
+			err := errors.ErrMySQLConnectionClosed
+			return test{
+				name: "return error when MySQL connection is closed",
+				args: args{
+					ctx: ctx,
+					val: "vald-01",
+				},
```

Comment:
```suggestion
			err := errors.ErrMySQLConnectionClosed
			return test{
				name: "return error when MySQL connection is closed",
				args: args{
					ctx: context.Background(),
					val: "vald-01",
				},
```

Reply:
@hlts2 could you please tell me why you think that?

Reply:
I didn't handle context.Done in this function, so I think it wasn't necessary :thinking: